install/bootstrap_livecd_usb.md

```diff
@@ -56,10 +56,16 @@ Fetch the base installation CSM tarball and extract it, installing the contained
 
    The ISO and other files are now available in the directory from the extracted CSM tar.
 
+1. Configure zypper with the `embedded` repository from the CSM release.
+
+   ```bash
+   linux# zypper ar -fG "${CSM_PATH}/rpm/embedded" "${CSM_RELEASE}-embedded"
+   ```
```

Comment:
Why not do this for the livecd path too?

Reply:
This was adding a repository to the linux machine to build the USB, which is fine. However we don't want to add the repository to the pit, at least not necessarily because we have 0 defined there as is. Contexts are different between this and the other page at this point in the install.